Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -476,6 +481,10 @@ const DEFAULT_VALUES = {
   previousSelectedData: null,
   widgetInFocus: null,
   captureOn: CaptureOn.MOUSE_MOVE,
+  cursorStyles: {
+    default: 'default',
+    hover: 'pointer',
+  },
```

Comment:
you must use the `defaultValues()` function notation if you have an object as value, otherwise it will be shared between ALL the widget manager instances.

Reply:
Thanks for catching that, fixed.